### DISCUSSION TITLE
VRoid SDK0.2.1 リリースのお知らせ

### DISCUSSION THREAD

**Discussion by ytomi-pixiv** [pixiv/vroid-sdk-developers#83] [Announcements]
SDK開発者のみなさま

VRoid SDK0.2.1 をリリースしました。

MultiplayExampleにおいて一部UI画像が読み込めていなかった不具合などに対応しました。

## CHANGELOG

- MultiplayExample.unityにおいてVRM 1.0モデルのライセンスが表示できなかった不具合に対応
- MultiplayExample.unityにおいて殆どのUI画像が読み込めていなかった不具合に対応

**Comment by FujiSunflower** [comment]
UniVRM側での不具合ですので単に連絡となりますが、VRM 0系のファイルを読み込んだ際のSpringBoneに不具合があるようです。
https://github.com/vrm-c/UniVRM/issues/2048

**Comment by RearAlice** [comment]
これは、Multiplayのみの修正でしょうか？
それともSDK全体のアップデートでしょうか？

**Reply by nyamadan** [reply]
MultiplayとUploadのみの修正となっています。
